docs/HowToUseAudio.md

```diff
@@ -53,9 +55,15 @@ After setting up audio codec, the codec's lowerhalf is wrapped around in an audi
 Psuedocode
 
 ```
-	/*Embed codec device witin audio device */
+	/*Embed codec device within audio device */
 	ret = audio_register(devname, codec_lowerhalf);
 ```
 
 Example: [os/arch/arm/src/artik05x/src/artik055_alc5658.c](../os/arch/arm/src/artik05x/src/artik055_alc5658.c)
 
+
+Driver registering policy (*devname* Naming)
+
+>Input: 	/dev/pcmC[card id]D[device id]c
```

Comment:
Let's remove below empty line and add two spaces at end of this line.
That is better to see.

Reply:
Modified.
Thank you for good guides.